<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardReader::services::RacomClientWidget</class>
 <widget class="QWidget" name="CardReader::services::RacomClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Racom Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Network communication</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkListenRawData">
        <property name="text">
         <string>Raw SI data</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Listen on UDP port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="edRawDataListenPort">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkListenSirxdData">
        <property name="text">
         <string>sirxd data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Listen on TCP port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="edSirxdDataListenPort">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1025</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbReadSplitsFile">
     <property name="title">
      <string>Read Text Splits File (format &quot;rawsplits&quot; from Racom)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>File Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Finish code (in file)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="edReadSplitFinishCode">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Import interval</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="edReadSplitInterval">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="edReadSplitsFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="btChooseSplitsFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkListenRawData</tabstop>
  <tabstop>edRawDataListenPort</tabstop>
  <tabstop>chkListenSirxdData</tabstop>
  <tabstop>edSirxdDataListenPort</tabstop>
  <tabstop>gbReadSplitsFile</tabstop>
  <tabstop>edReadSplitsFile</tabstop>
  <tabstop>btChooseSplitsFile</tabstop>
  <tabstop>edReadSplitFinishCode</tabstop>
  <tabstop>edReadSplitInterval</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
